mea/src/mutex/mod.rs

```diff
@@ -168,7 +176,7 @@ impl<T: ?Sized> Mutex<T> {
     /// use mea::mutex::Mutex;
     ///
     /// let mutex = Mutex::new(1);
-    /// let mut guard = mutex.try_lock().expect("mutex is locked");
+    /// let mut guard = mutex.try_lock().expect("mutex should be available");
```

Comment:
Unrelated changes? Ditto others.

Looks like something vibe coding touched. Please try to avoid this kind of change unless you have a good reason.